kotsadm/pkg/apiserver/bootstrap.go

```diff
@@ -96,7 +93,6 @@ func bootstrapIdentity() error {
 			return errors.Wrapf(err, "failed to get stat identity config file for app %s", app.Slug)
 		}
 
-		decode := scheme.Codecs.UniversalDeserializer().Decode
```

Comment:
any reason this was moved 10 lines above? :)

Reply:
grouping... :srug: